proto/greenfield/payment/delayed_withdrawal_record.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package greenfield.payment;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/bnb-chain/greenfield/x/payment/types";
+
+message DelayedWithdrawalRecord {
+  // timestamp is the unix timestamp to record the start of the withdrawal
+  int64 timestamp = 1;
```

Comment:
should we add an end timestamp for the record, so the user do not need to calculate later

Reply:
will only record the `unlock_timestamp`.